app/controllers/ListEventsController.js

```diff
@@ -9,10 +10,21 @@ GameUpApp.controller('ListEventsController', ['$scope', '$location', function($s
     return query.find();
   };
 
+  $scope.retrievePromotedEventList = function(){
```

Comment:
You can set limits on your Parse query
```query.limit(10); ```
If you don't explicitly set a limit, the default is 100, I think.

Also, fetching events by region could be done as follows:
```
var point = new Parse.GeoPoint({latitude: 40.0, longitude: -30.0});
query.near("location", point);
```
We can set the "location" attribute to Toronto's latitude & longitude by default